packages/atomic/src/pages/examples/commerce-website/search.html

```diff
@@ -94,7 +94,7 @@ <h1>Search page</h1>
                         <atomic-product-price currency="USD"></atomic-product-price>
                       </atomic-product-section-emphasized>
                       <atomic-product-section-description>
-                        <atomic-product-description></atomic-product-description>
+                        <atomic-product-text field="excerpt" class="block text-neutral-dark"></atomic-product-text>
```

Comment:
Why not use atomic-product-description and have it use atomic-product-text internally? We lose the expandable description otherwise...

Reply:
I do think it makes a lot of sense to also allow using the excerpt in `atomic-product-description`, good idea!